pkg/service/markdown.go

```diff
@@ -41,10 +43,14 @@ func NewMarkdownService(referenceURL string) (MarkdownService, error) {
 			})
 		}
 	}
+	if apiName != "" {
+		apiName = "/" + apiName
```

Comment:
Usually string concatenation in go works better using `fmt.Sprintf(<format>, <vars>)` as normal concatenation creates new strings per each variable. In this case this will create a string for `apiName` then a string for `/` and then a string for the result, which results in non efficient code (in this case has no big impact but I think is good to get used to avoid this string concatenation)

Reply:
```suggestion
		apiName = fmt.Sprintf("/%s", apiName)
```

Reply:
@afdecastro879 I tried to use the suggested changes feature here to make the change, but I cannot "commit suggestion", are you able to?

Reply:
Nevermind, I can commit this change.